docs/reference/topology-updater-commandline-reference.md

```diff
@@ -21,7 +21,8 @@ To quickly view available command line flags execute `nfd-topology-updater -help
 In a docker container:
 
 ```bash
-docker run gcr.io/k8s-staging-nfd/node-feature-discovery:master nfd-topology-updater -help
+docker run gcr.io/k8s-staging-nfd/node-feature-discovery:master \
```

Comment:
nit: a bit unrelated but now that you touch it we could update it to "dynamically" always use the correct image

```suggestion
docker run {{ site.container_image }} \
```